### PR TITLE
1682master

### PR DESCRIPTION
**Pull request by mjburton11**
@whoburg: file `gas_male_rubber.py` ready for review.  

I still have some work to do.  (creating make file/md file)  But I finish the most important part which was updating the main model in `gas_male_rubber` to incorporate a mission class and account for cases with and without a fixed engine.  The answers are nearly the same as in the CDR with slight differences due to some upgrades. 

NOTE: you have to checkout `82fixes` in `gpkit` for this to run.

### CODE REVIEW

File: 1682/gas_male/gasmale.md [hunk 1]
```diff
@@ -172,13 +172,13 @@ for var, fig in zip(altitude_vars, figs):
...
 
-By further discritizing the climb, cruise, and loiter mission segments the following figures were generated to follow the performance over the duration of the mission. 
+By further discretizing the climb, cruise, and loiter mission segments the following figures were generated to follow the performance over the duration of the mission. 
 
 ```python
 #inPDF: skip
 from plotting import plot_mission_var
 
-Mprof = GasMALE(DF70=True, discrit=True)
+Mprof = GasMALE(DF70=True, discrete=True)
```
Comment (whoburg) [inline 1682/gas_male/gasmale.md]: 👍 I had to fix this locally today...

**Comment by mjburton11** [comment]
@whoburg ready for review/squashing

I just went through and tagged all the commits that I thought were relevant so you could squash them.  A lot of the earlier commits from Feb and March I wasn't even sure what was going on so I kind of just let them be...

**Comment by mjburton11** [comment]
@bqpd can I delete this branch now?

**Comment by bqpd** [comment]
Yup! I'll do it right now

**Comment by bqpd** [comment]
oops, looks like another PR depends on it?

**Pull request closed without merging**